app/js/filterFunctions.ts

```diff
@@ -6,6 +6,10 @@ document.getElementById('testAllocated').addEventListener('change', () => {
     updateScoreTable()
 })
 
+document.querySelector('#categoryFilter').addEventListener('change', () => {
+    updateScoreTable()
```

Comment:
you dont need to call an anonymous function with a named function inside, just pass the named one into the callback

Reply:
done

Reply:
i think

Reply:
changed back as not possible